doc/stdenv/platform-notes.chapter.md

```diff
@@ -1,5 +1,20 @@
 # Platform Notes {#chap-platform-notes}
 
+## UEFI {#sec-uefi}
+
+Some common issues when packaging software for UEFI environments:
+
+- Compilers don't know about UEFI as a proper target, except recently in LLVM (https://reviews.llvm.org/D131594).
+
+  Most software will offer ways to compile UEFI software as it was compiled for the host system (not the UEFI host system) which will either confuse nixpkgs, you or will be a hack.
+
+  Examples of such software to learn about how they do it are GNU GRUB (GNU-EFI), systemd-boot (elf2efi) and EDK2 firmware (bespoke build scripts)
+
+- Nixpkgs cannot know if you have UEFI firmware support or not on your platform beyond what is declared to be supported.
+
+  Use `pkgs.stdenv.hostPlatform.hasEfi` to know if it potentially have any form of UEFI support.
+  Use `pkgs.stdenv.hostPlatform.isEfi` to know if you are building a software to run on the UEFI environment (i.e. without any operating system).
```

Comment:
Where is this defined?

Reply:
`isEfi` ? in `lib/systems/inspect.nix` ?

Reply:
[Where?](https://github.com/NixOS/nixpkgs/blob/b80e6c98e42db3187e8115ec31b3f0f0f5126f0b/lib/systems/inspect.nix)  You just renamed it, so there's no more definition of `isEfi`.

Reply:
Apologies, I lost myself in my own changes. I will remove this line because I am not sure it's consensual.

Reply:
Removed.